docker-compose.yml

```diff
@@ -15,18 +15,18 @@ services:
       - "7777:7777"
     environment:
       - RUN_DEV_SERVER=True
-      - PORT=7777
-      - HOST=0.0.0.0
       - DEBUG=True
+      - PROTOCOL=https
```

Comment:
Can we just detect if RUN_DEV_SERVER is true or not and use http based on that? I think that will reduce the amount of stuff we have to configure.